.pre-commit-config.yaml

```diff
@@ -19,6 +19,7 @@ repos:
   rev: v2.0.0
   hooks:
   - id: setup-cfg-fmt
+    args: [--include-version-classifiers, --max-py-version=3.11]
```

Comment:
This got missed recently, you now need `--include-version-classifiers` as of setup-cfg-fmt 2.